_test/test_tobyfit/test_tobyfit.m

```diff
@@ -0,0 +1,560 @@
+classdef test_tobyfit < TestCaseWithSave
```

Comment:
Couple of general comments. 

1. It's good that this has been fitted into the TestCase framework, but it is a very thorough refactor. Are we certain that the tests are unchanged in function? How could we verify?
2. There have been changes in the .mat files, which are binary so opaque. It would be good if the changes (which should effectively be null changes) are documented in the  PR description. I would expect that repackaging the saved objects to different names would not be a problem, but data changes might need further scrutiny.

Reply:
I cannot prove the tests are unchanged beyond the fact that in creating the new object, they were copied and pasted with the visual/legacy checks stripped out, however, I do believe that the only important factor is that the new test-suite is deemed to thoroughly test the aspects of the Tobyfit stuff.

There have been no changes to the data (though I would like to at some point to reduce the 87MB Rb test data size), only repackaged to combine the 4 existing test files into two simpler modules.